src/align.py

```diff
@@ -16,50 +16,95 @@ def get_edits(p: str, q: str) -> tuple[str, str, str]:
 
     """
     assert len(p) == len(q)
-    # FIXME: do the actual calculations here
-    return '', '', ''
 
+    CIGAR = ''
+    for i in range(len(p)):
+        if p[i] != '-' and q[i] != '-':
+            CIGAR += 'M'
+        if p[i] == '-' and q[i] != '-':
+            CIGAR += 'I'
+        if p[i] != '-' and q[i] == '-':
+            CIGAR += 'D'
 
-def local_align(p: str, x: str, i: int, edits: str) -> tuple[str, str]:
+    if len(p) == 0 and len(q) == 0:
+        p_out = ''
+        q_out = ''
+        CIGAR = ''
+    else:
+        p_out = p.replace('-','')
+        q_out = p.replace('-','')
+        CIGAR = CIGAR
```

Comment:
I think that if you get here, you will read `CIGAR` before you have written to it, which will be an error.